pkg/sync/v7/translate.go

```diff
@@ -352,6 +352,41 @@ var translations = map[string][]struct {
 			Template: "{{ .Config.GenevaLoggingControlPlaneRegion }}",
 		},
 	},
+	"DaemonSet.apps/openshift-azure-logging/omsagent": {
+		{
```

Comment:
by setting this label to true, we cause the sync pod to create the resource. when it is set to false, sync pod will delete it.